src/lib/util.js

```diff
@@ -91,3 +94,7 @@ const comp = (exports.comp = (...args) => {
  * Convert a timestamp into an ISO date string.
  */
 exports.iso = time => new Date(time * 1000).format('iso');
+
+exports.fromJS = o => fromJS(o);
```

Comment:
Is there a particular reason for doing this?
Why not directly requiring wherever we need it `fromJS` from `immutable`.

Reply:
No good reason, will remove.